index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+	<link href='https://fonts.googleapis.com/css?family=Roboto:400,700,900,300' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<div class="container">
+
+		<!-- PRODUCT 1 ARTICLE -->
+
+		<div class="card">
```

Comment:
The instructions state each card should be an article, yours are each a div, _containing_ an article.  Close, but not quite.